src/Items/ItemItemFrame.h

```diff
@@ -34,7 +34,6 @@ class cItemItemFrameHandler :
 
 		AddFaceDirection(a_BlockX, a_BlockY, a_BlockZ, a_BlockFace);  // Make sure block that will be occupied is free
 		BLOCKTYPE Block = a_World->GetBlock(a_BlockX, a_BlockY, a_BlockZ);
-		AddFaceDirection(a_BlockX, a_BlockY, a_BlockZ, a_BlockFace, true);  // We want the clicked block, so go back again
```

Comment:
I see this is "countered" by the protocol code, but what about the data saved to the chunk, is it valid then?

Reply:
Wiki says Vanilla changed storage to the block it's in back in 1.8, the last one we support.